controller/src/controller/config.py

```diff
@@ -498,6 +499,135 @@ class EnabledFileserverConfig(FileserverConfig):
     ] = []
 
 
+class FSAdminConfig(BaseModel):
+    """Configuration for filesystem administration environment."""
+
+    model_config = ConfigDict(alias_generator=to_camel, populate_by_name=True)
+
+    affinity: Annotated[
+        Affinity | None,
+        Field(
+            title="Affinity rules",
+            description="Node and pod affinity rules for file server pods",
+        ),
+    ] = None
+
+    application: Annotated[
+        str | None,
+        Field(
+            title="Argo CD application",
+            description="Argo CD application into which to put fsadmin pods",
+        ),
+    ] = None
```

Comment:
This should always be in the same application as the controller, not independently configurable, since it's using the controller's namespace.